airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, config: Mapping[str, Any], stream_name: str = None, param: Ma
         self.config = config
         self.start_date = config["start_date"]
         self.window_in_days = config["window_in_days"]
-        self._url_base = config["domain_url"]
+        self._url_base = config["domain_url"].rstrip("/") + "/"
```

Comment:
what is the purpose of the rstrip portion?

Reply:
The idea is that we're not sure if the value has trailing slash. This way allows us to be sure.